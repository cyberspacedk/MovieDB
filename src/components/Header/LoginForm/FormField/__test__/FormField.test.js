import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import FormItem from '../component';

describe('Component: FormItem', () => {
  const props = {
    type: 'text',
    name: 'username',
    placeHolder: 'Username',
    value: {
      username: 'Sam',
    },
    errors: {},
    touched: {},
  };

  it('should match its snapshot.', () => {
    const UserInfoComponent = shallow(<FormItem {...props} />);
    expect(shallowToJson(UserInfoComponent)).toMatchSnapshot();
  });
});
