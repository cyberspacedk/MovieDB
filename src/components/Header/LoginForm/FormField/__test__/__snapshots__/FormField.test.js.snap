// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: FormItem should match its snapshot. 1`] = `
<styled.div>
  <FormikConnect(FieldInner)
    name="username"
    placeholder="Username"
    type="text"
    value="Sam"
  />
</styled.div>
`;
